deepchem/models/tests/test_gnn3d.py

```diff
@@ -69,3 +77,20 @@ def test_net3d():
     output = net3d(graph)
 
     assert output.shape[1] == target_dim
+
+
+@pytest.mark.pytorch
+def test_InfoMax3DModular():
+    from deepchem.models.torch_models.gnn3d import InfoMax3DModular
+
+    data, _ = get_regression_dataset()
+
+    model = InfoMax3DModular(hidden_dim=64,
+                             target_dim=10,
+                             aggregators=['sum', 'mean', 'max'],
+                             readout_aggregators=['sum', 'mean'],
+                             scalers=['identity'])
+
+    loss1 = model.fit(data, nb_epoch=1)
+    loss2 = model.fit(data, nb_epoch=9)
+    assert loss1 > loss2
```

Comment:
Do we have save/reload tests for this model? 

The loss reduction here handles basic functionality, but should we add an overfit test?